packages/frontend/src/components/common/Filters/FieldLabel.tsx

```diff
@@ -5,27 +5,33 @@ import {
     isField,
     TableCalculation,
 } from '@lightdash/common';
+import { Text } from '@mantine/core';
 import { FC } from 'react';
-import styled from 'styled-components';
 
 interface FieldLabelProps {
     item: Field | TableCalculation | AdditionalMetric;
 }
 
-const BolderText = styled.span`
-    font-weight: 600;
-`;
-
 const FieldLabel: FC<FieldLabelProps> = ({ item }) => {
     return (
-        <span>
+        <Text span>
             {isField(item) ? `${item.tableLabel} ` : ''}
-            <BolderText>
+
+            <Text span fw={500}>
                 {isField(item) || isAdditionalMetric(item)
                     ? item.label
                     : item.displayName}
-            </BolderText>
-        </span>
+            </Text>
+        </Text>
+    );
+};
+
+export const fieldLabelText = (item: FieldLabelProps['item']) => {
+    return (
+        (isField(item) ? `${item.tableLabel} ` : '') +
+        (isField(item) || isAdditionalMetric(item)
+            ? item.label
+            : item.displayName)
```

Comment:
unsure about: `isAdditionalMetric(item)` here. At some point `additionalMetric`s get a `fieldType` `metric` added. Could you check?

Reply:
im basically copying the logic above from the component which makes sense.
we are looking for a proper label for an item (in this case it can be Field, Metric, AdditionalMetric or a TableCalculation)
all Field, Metric and AdditionalMetric except TableCalculation provides `label` property

![CleanShot 2023-06-27 at 16 16 28@2x](https://github.com/lightdash/lightdash/assets/962095/68e0829c-8a06-4092-9a8b-dd1bab393e8d)

Reply:
this can be `isField` || `isMetric` || `isAdditionalMetric` but since metric extends Field it does not make sense to further narrow down the type